assemblers/tcp_reader.go

```diff
@@ -108,7 +108,7 @@ func (reader *tcpReader) reassembledSG(sg reassembly.ScatterGather, ac reassembl
 			res.Body.Close()
 		}
 
-		if entry, matchFound := reader.matcher.GetOrStoreResponse(requestId, ctx.CaptureInfo.Timestamp, res); matchFound {
+		if entry, matchFound := reader.matcher.GetOrStoreResponse(requestId, ctx.CaptureInfo.Timestamp, res, sg.Stats().Packets); matchFound {
```

Comment:
What do you think of moving the packet count into a variable above so it's a little clearer what `sg.Stat().Packets` is?

```suggestion
		// get the number of packets that made up this response
		packetCount := sg.Stats().Packets
		if entry, matchFound := reader.matcher.GetOrStoreRequest(requestId, ctx.CaptureInfo.Timestamp, req, packetCount); matchFound {
```

Reply:
Good idea, done! https://github.com/honeycombio/honeycomb-network-agent/pull/262/commits/e0d1ac066c9249b7b85d11c9c4d9ef52025d8f1c